source/reference/logging.txt

```diff
@@ -0,0 +1,50 @@
+.. _scala-logging:
+
+=======
+Logging
+=======
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: record logs, log types
+
+By default, logging is enabled via the popular `SLF4J
+<https://www.slf4j.org/>`__ API. Logging is optional, so the driver
+uses SLF4J if the driver detects the presence of SLF4J API (class
+``org.slf4j.Logger``) in the classpath.
+
+Otherwise, the driver logs a single warning via JUL
+(``java.util.logging``) and logging is otherwise disabled.
+
+The driver uses the following logger names:
+
+- ``org.mongodb.driver``: the root logger
+  
+  - ``cluster``: for logs related to monitoring of the MongoDB servers to
```

Comment:
Question:
I haven't checked the source material, but should these be nested lists? It wasn't clear based on the introductory text. If it is, I'm ok with leaving it as-is.

Reply:
I kept the list formatted how it is in the original doc, I assume that the nested list is because the log message contains the root logger followed by combinations of the other log messages.